packages/user-group-service/src/users/resolver.ts

```diff
@@ -11,19 +11,19 @@ export const UserResolver = {
         } );
     },
     getUsersBy ( root: any, args: any, ctx: any ) {
-      const cleanedInput = pick( args, ['uid', 'rhatUuid'] );
+      const cleanedInput = pick( args, ['uid', 'rhatUUID'] );
```

Comment:
The `rhatUUID` needs to be converted into `rhUuid` since that's the field name used in the db schema.
Or else the line below wont return the expected result.

Reply:
Done. Thanks for the catch!